### DISCUSSION TITLE
Can't see assets in list view

### DISCUSSION THREAD

**Discussion by Bilge** [community/community#6018] [Other Feature Feedback, Questions, & Ideas] [answered]
Absolutely useless for me, since I only use releases to attach files. This is a serious regression.

**Comment by Gakk** [comment]
Instead of hiding assets, maybe collapsing the assets lists on the release list would be better?
If so, see #6009.

**Comment by DAOWAce** [comment]
> This is a serious regression.

Definitely feels like this.  Can't see any part of this that's an improvement for someone just looking to browse the release list; only the opposite.

**Comment by MylesBorins** [comment, marked as answer]
This came in https://github.com/github/feedback/discussions/5962#discussioncomment-1426476 and we already have a scoped work item to fix this.

Thank you for the feedback!

**Reply by MylesBorins** [reply]
we shipped a fix this week FYI
